python/semantic_kernel/utils/settings.py

```diff
@@ -230,3 +230,18 @@ def redis_settings_from_dot_env() -> str:
     ), "Redis connection string not found in .env file"
 
     return connection_string
+
+def azure_aisearch_settings_from_dot_env() -> Tuple[str, str]:
+    """
+    Reads the Azure AI Search environment variables for the .env file.
+
+    Returns:
```

Comment:
I'd love to please include exactly what is returned (e.g., the Azure AI Search API Key and the Azure AI Search URL).

Reply:
Makes sense as well, updated in commit 957f3d7d52d05075b1c524f32c5e037713dc73ae